stac/disasters/hurricane-harvey/hurricane-harvey-0831/Houston-East-20170831-103f-100d-0f4f-RGB/Houston-East-20170831-103f-100d-0f4f-RGB.json

```diff
@@ -16,6 +17,7 @@
     ],
     "constellation": "planetscope",
     "eo:cloud_cover": 2,
+    "pl:item_type": "PSScene3Band",
```

Comment:
I don't have metadata for it yet and assumed it's an old PSScene3Band. Thus the number of metadata entries is low. Can we improve this?

Reply:
See the other comment (removed pl metadata).